docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,65 @@
+## ISO norm requirements
 
-1. documents are uniquely identified;
-2. documents are approved for adequacy before issue by authorized personnel who have the =expertise and competence to determine adequacy;
-3. documents are periodically reviewed and updated as necessary;
-4. relevant versions of applicable documents are available at points of use and, where necessary, their distribution is controlled;
-5. changes and the current revision status of documents are identified;
-6. documents are protected from unauthorized changes and any deletion or removal;
-7. documents are protected from unauthorized access;
-8. the unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose;
-9. at least one paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements.
+To allow for a "Bioinformatician friendly" QMS, the following aspects need to be considered and documented :
 
-Examples are:
+1. Documents are uniquely identified
+2. Documents are approved for adequacy before issue by authorized personnel who have the expertise and competence to determine adequacy
+3. Documents are periodically reviewed and updated as necessary
+4. Relevant versions of applicable documents are available at points of use and, where necessary, their distribution is controlled
+5. Changes and the current revision status of documents are identified
+6. Documents are protected from unauthorized changes and any deletion or removal
+7. Documents are protected from unauthorized access
+8. The unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose
+9. One paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements
+
+## Why do Bioinformaticians need a different QMS
+
+Bioinformaticians create workflows collaboratively on a computer within a git environment. Git is a distributed version control system that tracks changes in any set of computer files. This can be used to track the changes of the code itself, but also in its documentation, which can describe the Standard Operating Procedures to use it. Git comes in with a set of features allowing to track issues and their resolution, new release of a workflow and its newest features. As the workflow often get upgraded and updated, their documentation, as well, is susceptible to change. Which is why many Bioinforaticians have turned their back from a standard QMS to have their own documentation system, often linked to git. Such system also offer possibilities to copy-paste code blocks while performing the SOP for a given flow, making it so much easier and less prone to errors. Examples of those documentations systems are:
```

Comment:
"..often linked to git." maybe add: "often linked to git, **code repositories and task management systems**."

Reply:
also add the reason that we want to be able to use specific formatting like codeblocks which is not possible in (most) QMS systems